### DISCUSSION TITLE
Big THANK YOU to everybody who is contributing to this project!

### DISCUSSION THREAD

**Discussion by CondorW** [cmderdev/cmder#2711] [General]
I come from Hyper Terminal and was constantly plagued with unstableness, bugs and slow performance. After trying out this tool, its like my development process has gotten so much smoother and more consistent.
I just really want to thank you guys for your effort - I love your work!

**Comment by DRSDavidSoft** [comment]
Big thanks to @chrisant996 for developing the Clink v1 fork, which is my favorite shell, and big thanks to @daxgames for developing and maintaining the very core of Cmder!! 😄 

You guys rock, and I appreciate all efforts you put into the project, as well the high-quality code that you write 😎 🤘

<h1 align=center>λ</h1>
